src/dotnet-structuring.library/Models/CustomProcess.cs

```diff
@@ -0,0 +1,25 @@
+﻿using dotnet_structuring.library.Interfaces;
+using System.Diagnostics;
+
+namespace dotnet_structuring.library.Models
+{
+    public class CustomProcess : Process, IProcess
```

Comment:
```suggestion
    public class TestProcess: Process, IProcess
```

And also define this only in the test project